okreplay-gradle-plugin/build.gradle

```diff
@@ -23,6 +23,8 @@ test {
   testLogging.showStandardStreams = isCi
 }
 
+/*
 validateTaskProperties {
   failOnWarning = true
 }
+ */
```

Comment:
Is this going to be reenabled at a later time?

Reply:
Good eye!  I disabled that when I went for broke trying to upgrade to AGP 7.x.  Now I can kick that can!